### ISSUE TITLE
populateFormValues is not filling Phone Number field value

### ISSUE THREAD

**Issue by romainpoirier** [verbb/formie#593]
**Description**
`populateFormValues` fails to fill Phone Number fields:
```
{% do craft.formie.populateFormValues(form, {
	myFormiePhoneNumberField: currentUser.myUserPhoneNumberField
}) %}
```

I can print the value of `currentUser.myUserPhoneNumberField`, which is for example `0412345678` (a valid Belgian phone number), but this value is never populated to the Phone Number field.

Maybe because this field is saving a split string (Country Code + Phone Number)?

### Form settings

- Multi-page form: Yes
- Submission Method: Page Reload
- Client-side Validation: Yes
- Custom Form Templates: Yes

**Additional info**
- Plugin version: 1.4.19
- Craft version: 3.7.14
- Multi-site: Yes

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.19",
  "...": "..."
}
```

Then run `composer update`. Also  see the updated [docs](https://verbb.io/craft-plugins/formie/docs/template-guides/populating-forms#phone)

**Comment by engram-design** [comment]
Fixed in [1.4.20](https://github.com/verbb/formie/releases/tag/1.4.20)
